validate.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+RUNS_PER_THREAD=100000
+MAX_PTHREADS=$(nproc)
+BIN_PATH="./build/bin"
+RES_FILE="res.txt"
+BUILD_LOG="build.log"
+STEP_SELETCTED=0
+STEP_THREADS=1
+FIBERS=""
+NUMA_NODES=""
+PASS_STATUS=1
+
+function usage()
+{
+    echo -e "Usage: ./validate.sh OPTION=NUM ...";
+    echo -e "This script compiles the sources in DEBUG mode and validates the correctnes of some of the provided concurrent objects.";
+    echo -e ""
+    echo -e "The following options are available."
+    echo -e "-t, --max_threads \t set the maximum number number of POSIX threads to be used in the last set of iterations of the benchmark, default is the number of system_cpus"
+    echo -e "-s, --step \t set the step (extra number of POSIX threads to be used) in succesive set of iterations of the benchmark, default is number of system_cpu/8 or 1"
```

Comment:
Maybe same here: `default is the number of system virtual cores/8 or 1`

Reply:
Done. I also fixed it for the case of bench.sh.